src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -481,6 +487,84 @@ private void RedirectToRelevantNode(string nodeTag)
             HttpContext.Response.Headers.Add("Location", location);
         }
 
+        [RavenAction("/databases/*/admin/backup", "POST", AuthorizationStatus.DatabaseAdmin, CorsMode = CorsMode.Cluster)]
+        public async Task BackupDatabaseOnce()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), "database-backup");
+                var backupConfiguration = JsonDeserializationServer.OneTimeBackupConfiguration(json);
+                PeriodicBackupRunner.CheckServerHealthBeforeBackup(ServerStore, backupConfiguration.Name);
+                ServerStore.LicenseManager.AssertCanAddPeriodicBackup(backupConfiguration);
```

Comment:
Also need to add here:
`AssertBackupConfiguration` and `AssertDestinationAndRegionAreAllowed`